api/python/t4/packages.py

```diff
@@ -316,12 +316,15 @@ def _from_path(cls, uri):
             raise NotImplementedError
         return pkg
 
-
-    def _clone(self):
-        """
-        Returns clone of this package.
-        """
-        return self.__class__(copy.deepcopy(self._data), copy.deepcopy(self._meta))
+    @classmethod
+    def _split_key(cls, logical_key):
+        if isinstance(logical_key, string_types):
+            path = logical_key.split('/')
```

Comment:
minor: we shouldn't we have a global `QUILT_SEPARATOR` or something. maybe proper URI handling will fix this.